Library/Formula/codeship-jet.rb

```diff
@@ -0,0 +1,18 @@
+class CodeshipJet < Formula
+  desc "Running Codeship Locally for Development"
+  homepage "https://codeship.com"
+  url "https://s3.amazonaws.com/codeship-jet-homebrew/0.11.5/jet-darwin_amd64.tar.gz"
```

Comment:
Alright. If it isn't permissively licensed you could always submit it to the [Caskroom](https://github.com/caskroom/homebrew-cask/).